.circleci/config.yml

```diff
@@ -169,24 +176,31 @@ workflows:
           filters:
             tags:
               only: /.*/
+      - licenses_test:
+          filters:
+            tags:
+              only: /.*/
       - integration_test_1:
           filters:
             tags:
               only: /.*/
           requires:
             - lint_unit_test_coverage
+            - licenses_test
```

Comment:
Doesn't this only need to be run once? Perhaps after unit tests?

Reply:
This is how it looks like: https://app.circleci.com/pipelines/github/dockstore/dockstore-ui2/2848/workflows/756bf50c-2c4e-4ba1-babf-2b46f6a9154b

Reply:
Yeah, this runs once and must pass before the unit tests are able to run.